x-pack/test_serverless/functional/test_suites/security/ftr/cloud_security_posture/mki_only/agentless/mki_create_agent.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { CLOUD_CREDENTIALS_PACKAGE_VERSION } from '@kbn/cloud-security-posture-plugin/common/constants';
+import expect from '@kbn/expect';
+import type { FtrProviderContext } from '../../../../../../ftr_provider_context';
+export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const pageObjects = getPageObjects([
+    'svlCommonPage',
+    'cspSecurity',
+    'security',
+    'header',
+    'cisAddIntegration',
+  ]);
+
+  const CIS_AWS_OPTION_TEST_ID = 'cisAwsTestId';
+
+  const AWS_SINGLE_ACCOUNT_TEST_ID = 'awsSingleTestId';
+
+  // This test suite is only running in the Serverless Quality Gates environment
+  describe('Agentless API Serverless', function () {
```

Comment:
The same suite title is already used for the local version of the test, which might cause confusion when looking into failed tests. I'd suggest to modify this suite title a little bit so it's easier to spot at glance which suite a test belongs to, e.g. something like `Agentless API Serverless MKI only`.